Makefile.build

```diff
@@ -316,11 +316,17 @@ $(LIBCOQRUN): kernel/byterun/coq_jumptbl.h $(BYTERUN)
 	cd $(dir $(LIBCOQRUN)) && \
 	$(OCAMLFIND) ocamlmklib -oc $(COQRUN) $(notdir $(BYTERUN))
 
-kernel/byterun/coq_jumptbl.h : kernel/byterun/coq_instruct.h kernel/byterun/make_jumptbl.sh
-	kernel/byterun/make_jumptbl.sh $< $@
+kernel/byterun/coq_instruct.h: kernel/genOpcodeFiles.ml
+	$(SHOW)'WRITE $@'
+	$(HIDE)(cd kernel/byterun && ocaml ../$(shell basename $<) enum)
+
+kernel/byterun/coq_jumptbl.h: kernel/genOpcodeFiles.ml
+	$(SHOW)'WRITE $@'
+	$(HIDE)(cd kernel/byterun && ocaml ../$(shell basename $<) jump)
 
-kernel/copcodes.ml: kernel/byterun/coq_instruct.h kernel/make_opcodes.sh kernel/make-opcodes
-	kernel/make_opcodes.sh $< $@
+kernel/copcodes.ml: kernel/genOpcodeFiles.ml
+	$(SHOW)'WRITE $@'
+	$(HIDE)(cd kernel && ocaml $(shell basename $<) copml)
```

Comment:
IIRC we were doing so for configure and ocaml would not abort on errors (just skip the broken toplevel sentence).
If it is the case, please use ocamlc first.